Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -38,23 +38,9 @@ private enum AssetCatalog {
   static let root = "com.apple.actool.catalog-contents"
   
   enum Extension {
-    static let appIconSet = "appiconset"
-    static let dataSet = "dataset"
-    static let gameCenterDashboardImage = "gcdashboardimage"
-    static let gameCenterLeaderboard = "gcleaderboard"
-    static let gameCenterLeaderboardSet = "gcleaderboardset"
-    static let iconSet = "iconset"
     static let imageSet = "imageset"
-    static let imageStack = "imagestack"
-    static let launchImage = "launchimage"
-    static let spriteAtlas = "spriteatlas"
-    static let stickerPack = "stickerpack"
-    
-    static let unsupported = [
-      appIconSet, dataSet, gameCenterDashboardImage, gameCenterLeaderboard,
-      gameCenterLeaderboardSet, iconSet, imageSet, imageStack, launchImage,
-      spriteAtlas, stickerPack
-    ]
+	
+    static let supported = [imageSet]
```

Comment:
Maybe just add a comment here to tell that this is the list of folder extensions in the `xcassets` folder that we know how to parse, so that in the future when people will want to add PRs to support more data type (I know I have a coworker wishing to add SpriteKit support soon for example) they won't forget to add them here.

Now that the whitelist is only one item (compared to the blacklist listing a lot of extensions) any contributor won't have a lot of examples to get inspiration from like we had with the blacklist version, so better documentation would help instead.